style.css

```diff
@@ -0,0 +1,16 @@
+#image {
+    display: block;
+    max-width: 100%;
+    height: auto;
+  }
+  
+  #img-div {
+    text-align:center;
+  }
+
+  h1 {
+    text-align: center;
+  }
+
```

Comment:
"text-align : center" appears more in your css file. Create css class so that you can reuse the in your html code. This will make your code look clean and DRY.